---
 binding_generator.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/binding_generator.py b/binding_generator.py
index 0bf46ea..23f329c 100644
--- a/binding_generator.py
+++ b/binding_generator.py
@@ -2,6 +2,7 @@
 
 import json
 import re
+import os
 import shutil
 from pathlib import Path
 
@@ -141,6 +142,7 @@ def print_file_list(api_filepath, output_dir, headers=False, sources=False):
 
 def scons_emit_files(target, source, env):
     files = [env.File(f) for f in get_file_list(str(source[0]), target[0].abspath, True, True)]
+    env.Append(ENV = os.environ)
     env.Clean(files, target)
     return [target[0]] + files, source
 
-- 
2.36.2

